### ISSUE TITLE
ncurl.exe: error: unrecognized arguments

### ISSUE THREAD

**Issue by insane-develop** [spulec/uncurl#59]
I copied curl from chrom, and then typed uncurl on the command line. It automatically read my clipboard, but doing so it reported an error.
uncurl
usage: uncurl.exe [-h] [-d DATA] [-b DATA_BINARY] [-X X] [-H HEADER] [--compressed] [-k] [--user USER] [-i] [-s] command url
 ncurl.exe: error: unrecognized arguments:

**Comment by atenaafshar** [comment]
yes seems there is an issue , in my case this is my original --data-raw : 
`'{"locale":"fa-IR","device_type":6,"version_code":2,"os_version":"macOS","device_name":"Chrome","referrer":0}'`
but seems uncurl escapes the quotes and turns it to : 
`'{locale:fa-IR,device_type:6,version_code:2,os_version:macOS,device_name:Chrome,referrer:0}'`
and it is not recognizable any more
